apps/typegpu-docs/src/content/examples/simple/vaporwave/floor.ts

```diff
@@ -0,0 +1,52 @@
+import tgpu from 'typegpu';
+import * as d from 'typegpu/data';
+import * as std from 'typegpu/std';
+
+import * as c from './constants.ts';
+
+export const grid = tgpu.fn(
+  [d.vec2f, d.f32],
+  d.vec3f,
+)((uv, time) => { // time is really an angle, but we are fine as long as it keeps increasing
+  const uvNormalized = std.fract(
+    std.div(d.vec2f(uv.x, uv.y + time), c.GRID_SEP),
+  );
+
+  // x^4 + y^4 = 0.5^4
+  const diff4 = std.pow(
+    std.sub(d.vec2f(0.5, 0.5), uvNormalized),
```

Comment:
```suggestion
    d.vec2f(0.5, 0.5).sub(uvNormalized),
```
If you merge 7.0, you should be able to use infix notation if you'd like to